lib/procfile.js

```diff
@@ -112,8 +112,8 @@ function readProcFile (pid, options, done) {
         cpu: cpu,
         memory: memory,
         ctime: (stat.utime + stat.stime) / cpuInfo.clockTick,
-        elapsed: date - (stat.start * 1000),
-        timestamp: stat.start * 1000, // start date
+        elapsed: (stat.uptime * 1000) - (stat.start * 1000), // process uptime minus start time
+        timestamp: date,
```

Comment:
I think that this should be the uptime of the process in ms since epoch but I'm not sure. @simonepri could you confirm? Thanks!